packages/shared/src/components/profile/ProfileForm.spec.tsx

```diff
@@ -37,24 +45,54 @@ const defaultUser = {
   createdAt: '2020-07-26T13:04:35.000Z',
 };
 
+const defaultFeedSettings: FeedSettings = {
+  advancedSettings: [{ id: 1, enabled: false }],
+  blockedTags: ['javascript'],
+  includeTags: ['react'],
+  excludeSources: [{ id: 'test', name: 'Source', image: 'a.b.c' }],
+};
+
+const createUpdateFeedFiltersMock = (
+  mock: ReturnType<typeof jest.fn>,
+  { advancedSettings, ...rest }: FeedSettings = defaultFeedSettings,
+) => {
+  const variables = { filters: rest, settings: advancedSettings };
```

Comment:
Perhaps we should include the alerts setting in this test as well?
What do you think?

Reply:
I tried but it fails, I think it is due to the fact we are mocking the parameters and the value was strongly typed.